regression/cbmc/Associativity1/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+CORE new-smt-backend
```

Comment:
Hi Michael, this is a good idea, and I've added it in my backlog, but if done it should be part of a separate PR (primarily because I want to keep this particular PR free of changes to CI)